chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/storemiddlewares/htmlTextMiddleware.ts

```diff
@@ -66,7 +67,7 @@ const processHTMLText = (action: IWebChatAction, text: string): IWebChatAction =
             }
 
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            action = (window as any).simpleUpdateIn(action, [Constants.payload, Constants.activity, Constants.text], () => htmlNode.innerHTML);
```

Comment:
we need to test this using CRM Livechat, also using popout

Reply:
yes tested popout mode too